main.py

```diff
@@ -55,6 +55,8 @@
     songname = d["data"]["songList"][0]["songName"]
     artistName = d["data"]["songList"][0]["artistName"]
     filename = "./" + songdir + "/"+songname+"-"+artistName+".flac"
+    if os.path.getsize(filename) < 10000 * 1024: #Source from http://stackoverflow.com/questions/8626325/most-efficient-way-to-delete-a-file-if-its-below-a-certain-size
```

Comment:
The position of this code is not right.
We should check the file size and delete it after downloading.